versioned_docs/version-latest/learn/monitoring.md

```diff
@@ -0,0 +1,723 @@
+---
+title: Monitoring
+sidebar_position: 4
+description: Monitoring server with Grafana and Prometheus. Scraping metrics from Subspace Node.
+keywords:
+  - Monitoring
+  - Grafana
+  - Prometheus
+  - Exporter
+  - Metrics
+  - Docker
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Link from '@docusaurus/Link';
+import styles from '@site/src/pages/index.module.css';
+
+
+-----------
+
+### Setting Up Monitoring Infrastructure
+
+**Infrastructure Setup:**
+
+Commence by installing and configuring exporters, such as Node Exporter, on the servers earmarked for monitoring. Occasionally, Prometheus client libraries are integrated into application code to reveal metrics at `<host>/metrics`, facilitating the monitoring of specific applications (e.g., Subspace node leverages this functionality). No alterations are necessary in the exporter configuration files on these servers; the default configurations suffice.
+
+Configure Prometheus to collect data from Node Exporter by specifying settings in the `prometheus.yml` file, often found in `/etc/prometheus`. This file defines remote server addresses where exporters operate as scraping targets for Prometheus to retrieve metrics. Exporters make metrics available via HTTP, accessible through HTTP requests to the `/metrics` endpoint. Typically, this communication occurs over port 9090 by default.
+
+Set up a Grafana server or any tool interacting with Prometheus to query metrics and craft visualizations. This process encompasses establishing connections to databases housing metrics, configuring dashboards, graphs, variables, and jobs to aptly represent the data.
```

Comment:
```suggestion
Setup a Grafana server or any tool interacting with Prometheus to query metrics and craft visualizations. This process encompasses establishing connections to databases housing metrics, configuring dashboards, graphs, variables, and jobs to aptly represent the data.
```

Reply:
got it. Corrected all 'set up' occasions